plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -409,6 +409,9 @@ private void testCorruptedTableLocation(String tableName, Path tableLocation, bo
         assertQueryReturnsEmptyResult("SELECT column_name, data_type FROM information_schema.columns WHERE table_schema = CURRENT_SCHEMA AND table_name LIKE 'bad\\_person\\_%' ESCAPE '\\'");
         assertQueryReturnsEmptyResult("SELECT column_name, data_type FROM system.jdbc.columns WHERE table_cat = CURRENT_CATALOG AND table_schem = CURRENT_SCHEMA AND table_name LIKE 'bad\\_person\\_%' ESCAPE '\\'");
 
+        // $history system table returns an empty result for corrupted table
+        assertQueryReturnsEmptyResult("SELECT * FROM \"" + tableName + "$history\"");
```

Comment:
Shouldn't it rather fail?
Has it been returning empty results also before the changes?

Reply:
Sorry, I somehow mixed up old with new behavior. Both old and new code throws the same exception now.